_includes/navbar.html

```diff
@@ -25,10 +25,10 @@
                             </a>
                         {% endif %}
 
-                        <ul class="dropdown-menu{% if forloop.last %} dropdown-menu-right{% endif %}" aria-labelledby="drop{{item.index}}">
+                        <ul class="dropdown-menu sub-navbar{% if forloop.last %} dropdown-menu-right{% endif %}" aria-labelledby="drop{{item.index}}">
```

Comment:
Is there a reason why we should introduce a new class here and not use the existing .dropdown-menu class? It's a bit confusing not we're now using these two classes in _menu.scss, when they're referring to the same element.

Reply:
@PaulineNemchak, since the _menu.scss is already under the 'Custom' folder, it's already clear from the fold architecture that these are custom styles :)

Reply:
Good point @lilyndk, we don't need to create a new class for that because it is the same element. That is the reason we have custom to adjust the classes that already exists, more class is more maintainability.